hack/verify-generated-json-codecs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+echo "===== Verifying Generated Ugorji JSON Codecs ====="
+
+if output=`find ${OS_ROOT}/vendor/k8s.io/kubernetes -name "types.generated.go" | grep -v "client-go"`
```

Comment:
I'm not sure why this was excluding client-go... we don't want generated types there either